examples/syntax_dependencies.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * Example code to call Rosette API to get entities from a piece of text.
+ **/
+require_once dirname(__FILE__) . '/vendor/autoload.php';
+use rosette\api\Api;
+use rosette\api\DocumentParameters;
+use rosette\api\RosetteException;
+
+$options = getopt(null, array('key:', 'url::'));
+if (!isset($options['key'])) {
+    echo 'Usage: php ' . __FILE__ . " --key <api_key> --url=<alternate_url>\n";
+    exit();
+}
+$syntaxDependenciesData = "Sony Pictures is planning to shoot a good portion of the new \"Ghostbusters\" in Boston as well.";
```

Comment:
The tag name is syntax_dependencies_data.  It does not adhere to the camelCase requirement.  This implementation will not work.